diff -uNr lua-lluv-0.1.9/src/lluv_handle.c lua-lluv-0.1.9_patched/src/lluv_handle.c
--- lua-lluv-0.1.9/src/lluv_handle.c	2018-05-06 12:08:26.000000000 +0200
+++ lua-lluv-0.1.9_patched/src/lluv_handle.c	2018-10-16 14:04:01.423779158 +0200
@@ -27,6 +27,7 @@
 #include "lluv_fs_poll.h"
 #include "lluv_process.h"
 #include <assert.h>
+#include <string.h>
 
 static const char* LLUV_HANDLES_SET = LLUV_PREFIX" Handles set";
 static const char* LLUV_HANDLE_NIL_UD = LLUV_PREFIX" nil ud";
